.ci_support/linux_64_cuda_compiler_version11.1numpy1.19python3.9.____cpython.yaml

```diff
@@ -23,14 +23,12 @@ docker_image:
 libprotobuf:
 - '3.16'
 mkl:
-- '2021'
+- '2020'
```

Comment:
Why did htis get downgraded?

Reply:
I'm not sure why conda-smithy did this.

Reply:
yeah, neither am i. this is why i started to build MKL 2020 again. Not sure if people changed their mind about hte migration.